core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -221,79 +238,83 @@ oppia.controller('LearnerDashboard', [
       // so that special cases can be handled while sorting subscriptions.
       var value = entity[$scope.currentSubscribersSortType];
       if ($scope.currentSubscribersSortType ===
-          SUBSCRIPTION_SORT_BY_KEYS.IMPACT) {
+          SUBSCRIPTION_SORT_BY_KEYS_AND_I18N_IDS.IMPACT.key) {
         value = (value || 0);
       }
       return value;
     };
 
-    $scope.sortFeedbackFunction = function(entity) {
-      return entity[$scope.currentFeedbackThreadsSortType];
-    }
+    $scope.sortFeedbackThreadsFunction = function(feedbackThread) {
```

Comment:
Still not sure what "function" this is referring to -- it's not returning a function. Do you mean something like: getValueOfFeedbackThreadSortKey()?

Same thing for the corresponding functions for other entities.